filters/src/main/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLogger.java

```diff
@@ -131,6 +131,7 @@ public RecordStreamLogger(final RecordStreamSource recordStreamSource) {
     valueTypeLoggers.put(ValueType.MESSAGE_CORRELATION, this::logMessageCorrelationRecordValue);
     valueTypeLoggers.put(ValueType.USER, this::logUsersRecordValue);
     valueTypeLoggers.put(ValueType.CLOCK, this::logClockRecordValue);
+    valueTypeLoggers.put(ValueType.AUTHORIZATION, record -> "");
```

Comment:
🤯 Could we maybe like having a generic one with toString?

Reply:
That's fair, I'll change it for the other empty loggers too